core/storage/azure_object_storage.go

```diff
@@ -229,36 +237,74 @@ func (aos *AzureObjectStorage) CopyObject(ctx context.Context, fromBucketName, t
 		return nil
 	}()
 
-	if _, err := blobCli.CopyFromURL(ctx, fromPathUrl, nil); err != nil {
-		return fmt.Errorf("storage: azure copy from url %w abort previous %w", err, abortErr)
+	if isDir != "directory" {
```

Comment:
As discussed, if path is a directory, blob copying will be failed and exited leading to backup operation failure. Because in azure, there is no concept of directories and only blobs exists. For copying, it will treat directory path as blob which doesn't exist and will throw error.